leakcanary-android-sample/detekt-baseline.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+<SmellBaseline>
+  <Blacklist></Blacklist>
+  <Whitelist></Whitelist>
+</SmellBaseline>
```

Comment:
do we need empty files?

Reply:
I believe we do, so it can update those later if we add more exclusions.
Will double-check it.

Reply:
I've got a better plan! 
Instead of adding baseline files I would just annotate the existing issues with `@Suppress`. This makes more sense since they aren't the issues we will be fixed in the future; they are completely ok the way they are.

This way there will be no baseline files at all. What do you think?